### ISSUE TITLE
User authentication

### ISSUE THREAD

**Issue by lpatmo** [codebuddies/serverless-concept#2]
My guess is https://docs.netlify.com/visitor-access/oauth-provider-tokens/ is the standard way to support user authentication. What are the alternatives?

**Comment by mr-rob0to** [comment]
Yes, this is one option (and based on the docs very easy to implement!). I think we should come to a consensus on what provider we want our users to authenticate with for the site. 

IMO, as a dev community, I feel it makes a ton of sense to use GitHub today.  We can also pull in GH user info, such as commits to the CB repo and display them right in the site/profile!

**Comment by lpatmo** [comment]
++ to GitHub -- though folks may request an email/password option too. And using `Identity` makes the most sense if it's the standard way. 

I like how easy Identity is to set up, but I noticed a couple of disadvantages with using it:
1/ We'll need to upgrade to a $45/mo pro plan (not covered under the open source plan) if we want to support `role-based` access control with JWT (e.g. having admins vs moderators vs normal users):
![image](https://user-images.githubusercontent.com/4512699/67172621-deb80780-f370-11e9-9759-4d9a2d72c989.png)
2/ Custom login branding isn't supported until we're under the $99/mo plan
3/ Identity is free up to 1000 active users. I'm not sure what "active" users mean -- logged-in users? I think at that level of activity we'd be able to cover the additional $99/mo with sponsors, but it's overhead at that size for sure.

![image](https://user-images.githubusercontent.com/4512699/67172790-8cc3b180-f371-11e9-9698-fcc2094a0e07.png)
